sdk/profiles/hybrid20200901/resourcemanager/authorization/armauthorization/constants.go

```diff
@@ -0,0 +1,15 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// DO NOT EDIT.
+
+package armauthorization
+
+const (
+	moduleName    = "hybrid20200901/armauthorization"
+	moduleVersion = "v1.0.0"
```

Comment:
I don't think this is correct. The module name and version should be declared at the module level, not per package. Will need codegen support.

Reply:
For profile, we only have one module for all the RPs. If we only declare the model info in the root level, then telemetry data will not include any sub-package info. It that OK or we have other guideline for sub-package telemetry data format?

Reply:
@jhendrixMSFT I have removed all the `moduleName` and `moduleVersion` from the sub-packages and introduce a root replacement. The side effect is that we need to export these two variables. Do you think it is OK?